src/conn.h

```diff
@@ -30,6 +30,8 @@ typedef struct conn_registry {
 	juice_agent_t **agents;
 	int agents_size;
 	int agents_count;
+	juice_cb_mux_incoming_t cb_mux_incoming;
+	void *mux_incoming_user_ptr;
```

Comment:
We can't do this at the moment because we need to not clean up the socket in release_registry when cb_mux_incoming is present, otherwise release_registry will shut down the listen socket.
If we move cb_mux_incoming to conn_mux, we need to add a method to tell release_registry not to clean up the socket, which may be more complicated.